### PR TITLE
Python package reorganization

### PR DESCRIPTION
**Pull request by slacrherbst**
This PR reorganizes the python sub-directories to be consistent with the new standards for rogue libraries in firmware repositories. 

Python modules are no longer conditionally included based upon the firmware modules loaded. I have also removed conflicting namespaces.

This is related to the following two PRs:

https://github.com/slaclab/ruckus/pull/109
https://github.com/slaclab/amc-carrier-core/pull/224

### CODE REVIEW

**Comment by ruck314** [comment]
image
Does cryo-det have an equivalent of a "pre-release" branch?  It seems like this PR should be target for merging to that branch.

**Comment by slacrherbst** [comment]
Yes I will set one up for testing this re-org.

**Comment by slacrherbst** [comment]
I created a pre-release to stage these changes during development.

File: firmware/python/CryoApp/AppCore.py [hunk 1]
```diff
@@ -18,19 +18,27 @@
 #-----------------------------------------------------------------------------
 
 import pyrogue as pr
+import AppTop
 
 from AppHardware.AmcCryo._amcCryoCore import *
```
Comment (ruck314) [inline firmware/python/CryoApp/AppCore.py]: I think this should be changed from a `import *` to `import packageName as aliasName`

Comment (slacrherbst) [reply firmware/python/CryoApp/AppCore.py]: Probably but I was trying to avoid changing too much.

**Comment by slacrherbst** [comment]
Closing as this out of date.

**Pull request closed without merging**